packages/static/src/tabs/item/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+export interface Props {
+  key: React.Key
```

Comment:
why do you need so many props and you proceed to handle only children?

Reply:
using for panelLess component <TabItem key={item.key} title={item.title}>